student.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+class Student < Person
+  def initialize(name, age, classroom)
+    super(name, age)
```

Comment:
- [ ] The `name` parameter should be optional always plus it also makes sense to inquire of the student's name also, so that means it should be after the compulsory ones (age and classroom).

  ```
  def initialize(classroom, age, name = 'Unknown')
    super(age, name)
  end
  ```